rcljava/src/main/java/org/ros2/rcljava/events/EventHandlerImpl.java

```diff
@@ -64,16 +64,19 @@
    * @param eventStatusFactory Factory of an event status.
    * @param callback The callback function that will be called when the event
    *     is triggered.
+   * @param removeCallback Callback that will be called when this object is being disposed.
    */
   public EventHandlerImpl(
       final WeakReference<ParentT> parentReference,
       final long handle,
       final Supplier<T> eventStatusFactory,
-      final Consumer<T> callback) {
+      final Consumer<T> callback,
+      final Consumer<EventHandler> removeCallback) {
```

Comment:
nitpick (take it or leave it): `disposeCallback` sounds more clear to me.

Reply:
fixed https://github.com/osrf/ros2_java/pull/16/commits/f1c70c3767399a7b7db7f80b28b303a0b60aef81